lib/coin_flip_demo.dart

```diff
@@ -0,0 +1,203 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'dart:math';
+
+final Random random = Random();
```

Comment:
nit: make these members of the State object.

Reply:
as it would be reasonable to test with a Random object with a fixed seed, etc.

Reply:
Done.